### PR TITLE
Update weight related parameters in relay

### PR DESCRIPTION
**Pull request by svyatonik**
depends both on #598 (code) and #659 (decisions) => draft

### CODE REVIEW

File: relays/substrate/src/messages_lane.rs [hunk 1]
```diff
@@ -117,3 +118,53 @@ where
...
+	fn select_delivery_transaction_limits_works() {
+		assert_eq!(
+			select_delivery_transaction_limits::<RialtoToMillauMessageLaneWeights>(
+				bp_rialto::max_extrinsic_weight(),
+				bp_millau::MAX_UNREWARDED_RELAYER_ENTRIES_AT_INBOUND_LANE,
+			),
+			// We don't actually care about these values, so feel free to update them whenever test
+			// fails. The only thing to do before that is to ensure that new values looks sane: i.e. weight
+			// reserved for messages dispatch allows dispatch of non-trivial messages.
+			(1024, 866_583_333_334),
```
Comment (HCastano) [inline relays/substrate/src/messages_lane.rs:167]: How often do you expect this test to break and need to be manually fixed?

Also, what do you consider "sane" values? Based off the comment `select_delivery_transaction_limits` I _think_ that anything under 50% for dispatch wouldn't be sane - but what about the max number of messages? I'd like more clarity around this documented in the code

**Review by HCastano — APPROVED** [review]

File: relays/substrate/src/messages_lane.rs (hunk 1 above)
Comment (svyatonik) [reply relays/substrate/src/messages_lane.rs:167]: > How often do you expect this test to break and need to be manually fixed?

Whenever weights or other related parameters are changed. We don't update weights often, so I guess it won't be often.

> Also, what do you consider "sane" values? Based off the comment select_delivery_transaction_limits I think that anything under 50% for dispatch wouldn't be sane - but what about the max number of messages? I'd like more clarity around this documented in the code

Number of messages should be larger than zero. If it has became much lower (no strict value - just some tendency) after some change, then it should get additional attention. E.g. when it drops from 1024 to, say, 100, it means that some significant change has been introduced.

Comment (svyatonik) [reply relays/substrate/src/messages_lane.rs]: I've added couple of asserts and docs

Comment (HCastano) [reply relays/substrate/src/messages_lane.rs]: Thanks for the clarifications!

**Pull request merged**